pkgs/charmbracelet/mods/registry.yaml

```diff
@@ -0,0 +1,24 @@
+packages:
+  - type: github_release
+    repo_owner: charmbracelet
+    repo_name: mods
+    description: AI on the command line
+    link: https://github.com/charmbracelet/mods
```

Comment:
```suggestion
```

When repo_owner and repo_name are set, you can omit `link`.
https://aquaproj.github.io/docs/develop-registry/registry-style-guide/#omit-the-setting-which-is-equivalent-to-the-default-value